pep-8000.rst

```diff
@@ -3,6 +3,7 @@ Title: Python Language Governance Proposal Overview
 Author: Barry Warsaw <barry@python.org>
 Status: Active
```

Comment:
```suggestion
Status: Final
```

The selection process has concluded and produced an (Active) result, PEP 13, and this PEP is now historical, so this should be updated accordingly (as PEP 8001 was).